src/aws_encryption_sdk/materials_managers/mpl/cmm.py

```diff
@@ -0,0 +1,142 @@
+"""Retrieves encryption/decryption materials from the MPL."""
+
+# These dependencies are only loaded if you install the MPL.
+try:
+    # pylint seems to struggle with this conditional import
+    # pylint: disable=unused-import
+    from aws_cryptographic_materialproviders.mpl.errors import AwsCryptographicMaterialProvidersException
+    from aws_cryptographic_materialproviders.mpl.models import (
+        AlgorithmSuiteIdESDK,
+        CommitmentPolicyESDK,
+        DecryptMaterialsInput,
+        DecryptMaterialsOutput,
+        EncryptedDataKey as MPL_EncryptedDataKey,
+        GetEncryptionMaterialsInput,
+        GetEncryptionMaterialsOutput,
+    )
+    from aws_cryptographic_materialproviders.mpl.references import ICryptographicMaterialsManager
+
+except ImportError:
+    pass
+
+from typing import List
+
+from aws_encryption_sdk.exceptions import AWSEncryptionSDKClientError
+from aws_encryption_sdk.identifiers import CommitmentPolicy
+from aws_encryption_sdk.materials_managers.mpl.materials import MPLEncryptionMaterials, MPLDecryptionMaterials
```

Comment:
Question: Should this import be in the try block?

This:  `aws_encryption_sdk.materials_managers.mpl.materials`

Reply:
Not in this case. This particular module will require the MPL to be installed to import it.